app/src/main/res/values/themes.xml

```diff
@@ -248,7 +248,7 @@
     </style>
 
     <style name="Theme.TextSecure.Dialog.MediaSendProgress" parent="@android:style/Theme.Dialog">
-        <item name="android:colorBackground">?attr/dialog_background_color</item>
+        <item name="android:background">@drawable/default_dialog_background</item>
```

Comment:
this sets background on every `View` so looks highly suspect.

There is only one usage though...

```
dialog = new AlertDialog.Builder(new ContextThemeWrapper(requireContext(), R.style.Theme_TextSecure_Dialog_MediaSendProgress))
                                  .setView(R.layout.progress_dialog)
                                  .setCancelable(false)
                                  .create();
```

and  the views in that layout are:

```
<RelativeLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:layout_width="wrap_content"
    android:layout_height="wrap_content"
    android:padding="@dimen/medium_spacing">

    <com.github.ybq.android.spinkit.SpinKitView
        style="@style/SpinKitView.Large.ThreeBounce"
        android:layout_width="wrap_content"
        android:layout_height="wrap_content"
        android:layout_centerInParent="true"
        app:SpinKit_Color="?android:textColorPrimary" />

</RelativeLayout>
```

so this just sets both of them to have `default_dialog_background` as a background. and they have the same dimensions, so you won't see the background of the `RelativeLayout`

I think it'd be more correct to use `android:windowBackground` but isn't hugely important atm.

Reply:
Yea this is the last one I need to properly test - am having a hard time finding it via the actual UI 😂 

If I find it and can confirm it wasn't actually broken then I'll change it back to what you had it as

Reply:
`MediaSendFragment#sendButton` ...

<img width="557" alt="Screen Shot 2023-06-06 at 5 46 43 pm" src="https://github.com/oxen-io/session-android/assets/9282178/c815f150-19a1-4cd5-a314-1f8be1d5912a">

but it only shows if it takes longer than 250ms...

`Util.runOnMainDelayed(progressTimer, 250);`

Reply:
Looks like it's actually busted with either approach (white on white) so have reverted this back to your change and updated the progress to be the accent colour:
<img width="146" alt="Screenshot 2023-06-06 at 6 28 22 pm" src="https://github.com/oxen-io/session-android/assets/15862619/6580de06-b25e-4695-9952-97e20ee55d67">